### ISSUE TITLE
Confused by Readme

### ISSUE THREAD

**Issue by michaelfuckner** [intel/perfmon-metrics#5]
Hi,

I am trying to debug an ICX System with perfmon metrics and there are some things I don't understand from the readme

1) File metrics_icx_perf.json has to be: icx_metric_perf.json?
2) path in kernel source tree <linux kernel source root dir>/perfmon-metrics/pmu-events/arch/x86/icelakex/ has to be <linux kernel source root dir>/tools/perf/pmu-events/arch/x86/icelakex/
3) the example app ./mlc ist the Intel Memory Latency Checker?
./perf  stat -M memory_bandwidth -a -- /root/Linux/mlc
Cannot find metric or group `memory_bandwidth'

4) How does https://download.01.org/perfmon/ICX/ compare to https://github.com/intel/perfmon-metrics/tree/main/ICX/events ?

Regards,
 Michael!

**Comment by calebbiggers** [comment]
Hi, 
Thanks for pointing out the issues in the readme.
1. Yes, this was a holdover from a past rename. I've updated the readme.
2. Yes, the path was incorrect. I've updated the readme.
3. Yes, the example workload is Intel Memory Latency Checker and is meant as an example only, where any test binary can be used. Additionally, 'memory_bandwidth' is just an example metric name. It is not present in our icx file. I've updated the readme to use an example metric that is present in that file.
4. The events present at 01.org and the events in this repo are the same.

Thanks,
Caleb
